tests/testsupport/broker.py

```diff
@@ -53,7 +56,52 @@ def fake_broker():
     # print('Setup broker')
     broker = FakeBroker()
 
-    yield broker
+    try:
```

Comment:
You're right, I didn't mean to commit that but it got caught up in the commit. For future reference from the [documentation](http://pytest.org/dev/yieldfixture.html) for anyone reading:

> With pytest fixture functions, the “after yield” part will always be invoked, independently from the exception status of the test function which uses the fixture.